src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3617,7 +3617,7 @@ QString ExportMusicXml::elementPosition(const ExportMusicXml* const expMxml, con
         const double pageHeight  = expMxml->getTenthsFromInches(expMxml->score()->styleD(Sid::pageHeight));
 
         const auto meas = elm->findMeasure();
-        IF_ASSERT_FAILED(!meas) {
+        IF_ASSERT_FAILED(meas) {
             return res;
```

Comment:
Not a fan of his IF_ASSERT_FAILED macro, it's not obvious what it might from reading it.
Maybe
I'd actually prefer just 
```
  if (ASSERT_TRUE(!meas)) {
}
```
Such that the macro didn't invert the condition.

Reply:
DEBUG, Not RelWithDebInfo?

Reply:
OK, that might be the problem then

Anyway, seems that bug was a screwup of mine when porting @julienripoche`s #9955 in my #10456
And yes, that strange inverted lugic is what threw me off.

Reply:
Mind to check ef1a2cf whether that needs to get inverted too? @cbjeukendrup ?